launchable/utils/commit_ingester.py

```diff
@@ -0,0 +1,57 @@
+from .git_log_parser import GitCommit
+from .http_client import LaunchableClient
+from typing import List, Dict, Optional
+from datetime import tzinfo
+import hashlib
+
+
+def _sha256(s: str) -> str:
+    return hashlib.sha256(s.encode('utf8')).hexdigest()
+
+
+def _format_tzinfo(tz: Optional[tzinfo]) -> int:
+    if not tz:
+        return 0
+    delta = tz.utcoffset(None)
+    if not delta:
+        return 0
+    return round(delta.total_seconds() / 60)
+
+
+def _convert_git_commit(commit: GitCommit) -> Dict:
+    changed_files = []
+    for changed_file in commit.changed_files:
+        cf = dict()
+        cf['linesAdded'] = changed_file.added
+        cf['linesDeleted'] = changed_file.deleted
+        cf['status'] = 'M'
+        cf['path'] = changed_file.path
+        cf['pathTo'] = changed_file.path
+        changed_files.append(cf)
+    parents = dict()
+    if len(commit.parents) > 0:
+        # We don't know which diff is for which parent. Use the first parent.
+        parents[commit.parents[0]] = changed_files
+        for parent in commit.parents[1:len(commit.parents)]:
+            parents[parent] = []
```

Comment:
[question]
is it correct that you are adding empty list here?

Reply:
Yes. All changed files are added to the first parent, and the rest of the commits gets an empty changed file list.